integr8ly/installation
# ON HOLD - DO NOT MERGE - INTLY-2029 Remove Enmasse Postgres backup job & alert

## Additional Information
https://issues.jboss.org/browse/INTLY-2029

## Verification Steps

1. Install as normal, specifying a var override of `backup_restore_install=true`
2. Verify the Cronjob for `enmasse-postgres-backup` was not created in the integreatly-backups namespace
3. Verify the Cronjob alert for `enmasse-postgres-backup` is not showing up in Prometheus

## Is an upgrade task required and are there additional steps needed to test this?
Yes,
covered in #593
